### DISCUSSION TITLE
Bleachbit features and contribution

### DISCUSSION THREAD

**Discussion by itsnotsaved** [bleachbit/bleachbit#1257] [Ideas]
Does bleachbit has below features for linux?

- Package cache clean
- Crash report clean
- Application logs clean
- Application cache clean

source https://github.com/oguzhaninan/Stacer (project seems more to be abandoned)
it has many feature but i only used cleaning things

**Comment by Emajor69** [comment]
Add a cleaner for Ungoogled Chromium
